cotainr/tests/util/test_answer_is_yes.py

```diff
@@ -0,0 +1,71 @@
+"""
+cotainr - a user space Apptainer/Singularity container builder.
+
+Copyright DeiC, deic.dk
+Licensed under the European Union Public License (EUPL) 1.2
+- see the LICENSE file for details.
+
+"""
+
+import pytest
+import cotainr.util
+
+
+class TestAnswerYes:
+    def test_empty(self, capsys, factory_mock_input, monkeypatch):
+        monkeypatch.setattr("builtins.input", factory_mock_input(""))
+        answer = cotainr.util.answer_is_yes("")
+
+        assert not answer
+
+    @pytest.mark.parametrize("outp", ["Test!", "Test\nTest", "Whoop!", "\n", "\nTest!"])
+    @pytest.mark.parametrize("inp", ["n", "nope", "no", "NO!", "Not gonna happen"])
+    def test_n(self, capsys, outp, inp, factory_mock_input, monkeypatch):
```

Comment:
Does this really need to run 5 * 5 = 25 tests to ensure that no matter how the user replies, the prompt printed before they have had the chance to answer is printed correctly? 😄 

(Consider this repeated for the other tests that have these `parametrize` cartesian combinations.)

Reply:
Done.